# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/VMFunctions.h
# Commit: fdbb1c215d78
# Full Hash: fdbb1c215d783a85c27d71ed8bb832690962c948
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2016-03-28 20:04:22
# Regressor Bug: 1165052
# File Overlap Count: 1
# Description:
#   Bug 1165052 - Part 4: Inline IsConstructor intrinsic. r=jandem
# ==============================================================================

diff -r 5e02cbf51e45 -r fdbb1c215d78 js/src/jit/VMFunctions.h
--- a/js/src/jit/VMFunctions.h	Wed Dec 23 12:46:35 2015 +0900
+++ b/js/src/jit/VMFunctions.h	Mon Nov 16 09:01:37 2015 +0900
@@ -733,6 +733,7 @@
 }
 
 bool ObjectIsCallable(JSObject* obj);
+bool ObjectIsConstructor(JSObject* obj);
 
 bool ThrowRuntimeLexicalError(JSContext* cx, unsigned errorNumber);
 bool BaselineThrowUninitializedThis(JSContext* cx, BaselineFrame* frame);